src/imagewriter.h

```diff
@@ -61,6 +63,14 @@ class ImageWriter : public QObject
     /* Returns true if src and dst are set */
     Q_INVOKABLE bool readyToWrite();
 
+    /* Returns true if running on a Raspberry Pi */
+    Q_INVOKABLE bool isRaspberryPiDevice();
+
+    /* Returns true if running on a Raspberry Pi */
```

Comment:
Copy'n'paste comment? (Seems unlikely that a function returning a `QString` will "return true" :wink: )